draft-ietf-tls-esni.md

```diff
@@ -1424,7 +1430,7 @@ consideration when reasoning about the privacy properties that ECH provides.
 
 ECH requires encrypted DNS to be an effective privacy protection mechanism.
 However, verifying the server's identity from the Certificate message,
-particularly when using the X509 CertificateType, may result in additional
+particularly when using the X.509 CertificateType, may result in additional
```

Comment:
This is not correct. Yes, they are X.509 certificates, but the name of the enum in RFC 8446 (S 4.4.2i) ix "X509"

```suggestion
particularly when using the X509 CertificateType, may result in additional
```